models/pipeline.js

```diff
@@ -27,6 +27,11 @@ const MODEL = {
         .description('Unique identifier for the application')
         .example('github.com:123456:master'),
```

Comment:
Could we just use the first value before the first colon in the `scmUri`? Would this value ever be different from `scmContext`?